framework/Metrics/metrics/STDReduction.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on 2017 September 12
+
+@author: Joshua Cogliati
+"""
+
+from .MetricInterface import MetricInterface
+from Metrics.metrics import MetricUtilities
+
+class STDReduction(MetricInterface):
+  """
+    Metric to compare two datasets using the PDF Common Area.
+  """
+  def __init__(self):
+    """
+      Constructor
+      @ In, None
+      @ Out, NOne
+    """
+    super().__init__()
+    # If True the metric needs to be able to handle (value,probability) where value and probability are lists
+    #self.acceptsProbability = True
+    # If True the metric needs to be able to handle a passed in Distribution
+    #self.acceptsDistribution = True
```

Comment:
Do you need to keep these lines?

Reply:
I'd like to keep it first, want to make it applicable to probability and value pairs